packages/p2p/src/components/composite-calendar/composite-calendar.jsx

```diff
@@ -0,0 +1,201 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Loadable from 'react-loadable';
+import { DesktopWrapper, InputField, MobileWrapper, useOnClickOutside } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { daysFromTodayTo, epochToMoment, toMoment } from '@deriv/shared';
+import CompositeCalendarMobile from './composite-calendar-mobile.jsx';
+import SideList from './calendar-side-list.jsx';
+import CalendarIcon from './calendar-icon.jsx';
+import { observer } from 'mobx-react-lite';
+import { useStore } from '@deriv/stores';
```

Comment:
could we also make sure the imports are sorted like this:
- React
- Imported libraries
- @deriv files
- Components
- scss file (if any)